models/models.go

```diff
@@ -97,14 +97,18 @@ func LoadConfigs() {
 
 func getEngine() (*xorm.Engine, error) {
 	cnnstr := ""
+	var Param string = "?"
```

Comment:
Good point, I didn't see this change till now. This is that is not a explicit change done by me. Let sync again with your develop branch.